src/components/page/mod.rs

```diff
@@ -70,15 +70,19 @@ pub fn page(props: &PageProperties) -> Html {
     html! (
         <div class="pf-v5-c-page" id={&props.id} role="main" tabindex="-1">
             <header class="pf-v5-c-masthead">
-                <span class="pf-v5-c-masthead__toggle">
-                    <Button
-                        r#type={ButtonType::Button}
-                        variant={ButtonVariant::Plain}
-                        {onclick}
-                    >
-                        <i class="fas fa-bars" aria-hidden="true" />
-                    </Button>
-                </span>
+                // If the sidebar is empty then the toggle button is not 
+                // shown
+                if !props.sidebar.clone().is_empty() {
```

Comment:
```suggestion
                if !props.sidebar.is_empty() {
```

I don't think it needs to be cloned.

Reply:
Yes indeed cloning is superfluous, I'm still struggling to learn certain concepts.
You're right, I didn't run cargo fmt, now I did but what's the difference?